README.md

```diff
@@ -16,7 +10,17 @@ This tells you if your jobs are done. And stuff like that.
  - Gets input / output dataset names
  - Works with piping, other unix nice things
 
-## Installation
+Like badges? We have the best badges:
+
+[![GitHub Project](https://img.shields.io/badge/GitHub--blue?style=social&logo=GitHub)](https://github.com/dguest/pandamonium)
+[![DOI](https://zenodo.org/badge/DOI/10.5281/zenodo.4019463.svg)](https://doi.org/10.5281/zenodo.4019463)
+
+[![PyPI version](https://badge.fury.io/py/pandamonium.svg)](https://badge.fury.io/py/pandamonium)
+[![Supported Python versions](https://img.shields.io/pypi/pyversions/pandamonium.svg)](https://pypi.org/project/pandamonium/)
```

Comment:
People generally put these right at the top for quick reference for things people really want to get at, but there's nothing _wrong_ with them here.

Reply:
yeah I had mixed feelings about this: on one hand I agree with you, on the other hand the first thing you see if you read this with `less` is a bunch of machine looking crap. I'll think about it since this isn't urgent.

Reply:
>  on the other hand the first thing you see if you read this with `less` is a bunch of machine looking crap

This is true. I think the question here is who is the intended audience for the `README` and where do you expect them to read it? For most projects that I work on I'm assuming the `README` is meant for new users to the project who have never been there before and or returning users who want quick references and links. I'm also assuming that for Python projects on PyPI people are only cloning the repo if they want to look around or contribute and so are not reading the `README` from their terminal or text editor but are instead reading it from PyPI or GitHub where it is rendered out.

Reply:
I think you make a good point. I could go either way. For now I'll leave it as this merge puts it because I'm lazy.